app/src/main/java/io/github/thibaultbee/srtplayer/MainActivityViewModel.kt

```diff
@@ -42,6 +44,7 @@ class MainActivityViewModel(application: Application) : AndroidViewModel(applica
         player.release()
     }
 
+    @OptIn(UnstableApi::class)
```

Comment:
Do we have to propagate the `UnstableApi` to the main activity?
Could you use directly OptIn in the `player` class?
I am just asking.

Reply:
This declaration is opt-in and its usage should be marked with @androidx. media3.common. util. UnstableApi or @OptIn(markerClass = androidx. media3.common. util. UnstableApi. class